vscripts/mp/_codecallbacks.gnut

```diff
@@ -256,6 +256,19 @@ void function CodeCallback_DamagePlayerOrNPC( entity ent, var damageInfo )
 		}
 	}
 
+	if (DamageInfo_GetWeapon(damageInfo) != null) {
+		if (ent.GetShieldHealth() == 0) {
```

Comment:
`GetShieldHealth()` returns float, so maybe change if to `if ( GetShieldHealth() > 0 ) { }` and swap bodies?